server/dataset_context.h

```diff
@@ -109,12 +109,16 @@ struct SqlExpressionDatasetContext: public SqlExpressionMldbContext {
     {
         return RowContext(row, params);
     }
+
+    virtual Utf8String 
+    doResolveTableName(const Utf8String & fullVariableName, Utf8String &tableName) const;
```

Comment:
It seems that the naming of the function is incorrect.  Are we trying to resolve a variable name or a table name?